src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/BulkEditInAppTitle/BulkEditInAppTitle.js

```diff
@@ -1,31 +1,78 @@
 import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
 
 import {
   Col,
   Row,
+  Label,
 } from '@folio/stripes/components';
 
 import css from './BulkEditInAppTitle.css';
+import { FINAL_ACTIONS } from '../../../../../constants';
```

Comment:
Please move this import to be before css

Reply:
Fixed